src/igvfd/types/software_version.py

```diff
@@ -46,6 +46,7 @@ def unique_keys(self, properties):
         'type': 'string',
         'description': 'A unique identifier for a software version.',
         'notSubmittable': True,
+        'uniqueKey': True
```

Comment:
I think you've accidentally included changes from https://igvf.atlassian.net/browse/IGVF-1550.